ionc/ion_binary.c

```diff
@@ -26,7 +26,7 @@ int ion_binary_len_uint_64(uint64_t value) {
 }
 
 // These field formats are always used in some context that clearly indicates the number of octets in the field.
-int ion_binary_len_int64(int64_t value) {
+int ion_binary_len_int_64(int64_t value) {
```

Comment:
We exported a different function name in the header (int_64 vs int64), and the style chosen here matches the existing style in the other functions.